js/common.js

```diff
@@ -1010,6 +1010,9 @@ var getTextWithoutDiacriticalMarks = function (text) {
  * @return {string}
  */
 var escapeMarkupText = function (text) {
+    if (!isNaN(text)) {
```

Comment:
Checking variable type directly is probably easier to read than a double negation.
```suggestion
    if (typeof(text) !== 'string') {
```